src/praisonai-code/praisonai_code/cli/commands/session.py

```
@@ -472,6 +476,138 @@ def session_show(
+    """Revoke a previously published transcript."""
+    output = get_output_controller()
+
+    share_path = _share_path(session_id)
+    revoked = False
+    if share_path.exists():
+        try:
+            share_path.unlink()
+            revoked = True
+        except OSError as e:
+            output.print_error(f"Failed to unshare session: {e}")
+            raise typer.Exit(1)
```

Ruff (0.16.0)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)